lib/greengrasscoreipc/model.ts

```diff
@@ -373,42 +580,56 @@ export interface InvalidRecipeDirectoryPathError {
 
 export interface ServiceError {
 
-    message?: string
+    message?: string,
+
+    context?: any
 
 }
 
 export interface CreateLocalDeploymentResponse {
 
+    /**
+     * The ID of the local deployment that the request created.
+     */
     deploymentId?: string
 
 }
 
-/**
- * *groupName*: The thing group name the deployment is targeting. If the group name is
-not specified, "DEFAULT" will be used
-*rootComponentVersionsToAdd*: Map of component name to version. Components will be added
-to the group's existing root components.
-*rootComponentsToRemove*: List of components that need to be removed from the group
-For instance if new artifacts were loaded in this request but recipe version did not change
-*componentToConfiguration*: Map of component names to configuration.
-*componentToRunWithInfo*: Map of component names to component run as info.
-*recipeDirectoryPath*: All recipes files in this directory will be copied over to the kernel package store.
-*artifactDirectoryPath*: All artifact files in this directory will be copied over to the kernel package store.
- */
 export interface CreateLocalDeploymentRequest {
 
+    /**
+     * The thing group name the deployment is targeting. If the group name is not specified, "LOCAL_DEPLOYMENT" will be used.
```

Comment:
Trivial: The removed comment stated that "DEFAULT" would be used if name isn't specified. Just calling it out in case this was a unintended change.